src/modules/tests/vehicle-checks/__tests__/vehicle-checks.reducer.spec.ts

```diff
@@ -29,4 +34,33 @@ describe('vehicle checks reducer', () => {
     const result = vehicleChecksReducer({}, new TellMeQuestionDrivingFault());
     expect(result.tellMeQuestionOutcome).toBe('DF');
   });
+
+  it('it should set the show me question details', () => {
+    const newQuestionPayload = {
+      showMeQuestionCode: 'S1',
+      showMeQuestionDescription: 'desc',
+      showMeQuestionShortName: 'name',
+    };
+
+    const result = vehicleChecksReducer({}, new ShowMeQuestionSelected(newQuestionPayload));
+    expect(result.showMeQuestionCode).toBe('S1');
+    expect(result.showMeQuestionDescription).toBe('desc');
+  });
+
+  it('it should update the show me question details', () => {
```

Comment:
Our tests never use it in the it, start with should. would remove it in the text for all 
(that sounds clear as mud - happy to clarify)

Reply:
clear as mud..my bad..